pyramid/tests/test_util.py

```diff
@@ -619,7 +634,30 @@ def test___str__(self):
                          "Line 0 of file filename:\n       linerepr  ")
 
 
+class TestCallableName(unittest.TestCase):
+    def test_valid_ascii(self):
+        from pyramid.util import get_callable_name
+        name = u'hello world'
```

Comment:
Yeah, I refuse to install that shitty version of python on my system and always forget :P